README.md

```diff
@@ -36,24 +36,7 @@ To use our metric in practice, we want to increment the counter when tracking ev
 You should add these `.Inc()` calls in the place in your code where the event you want to track is occuring.
 
 
-Labels:
-
-Labels are a way of adding contextual information to your metrics (increasing their "cardinality"). For example, when tracking the count of requests received, it might be useful to also track the status code of the request. To do this, you can use `prometheus.NewCounterVec()` instead of `prometheus.NewCounter()`, and provide a list of label keys as the second argument - for example:
-
-    requestCounter := prometheus.NewCounter(prometheus.CounterOpts{Name: "requests_total"}, []string{"status_code"})
-
-Then, when tracking it, you'll need to provide the label values, which can be done like so:
-
-    var status int
-    if something.wasSuccessful:
-        status = http.StatusOK
-    else:
-        status = http.StatusInternalServerError
-
-	requestCounter.WithLabelValues(status).Inc()
-
-
-### Scraping Metrics with Prometheus
+### Section 3: Scraping Metrics with Prometheus
 
 So far, we've been able to instrument our application, such that it is now exporting metrics about its runtime behaviour. However, we still need to collect those metrics and store the data in a way that we can query it back out, in order to graph it over time and make dashboards.
 
```

Comment:
Could add here that we now run with Docker & include command?

Reply:
I think it is, it's just hidden (folded) in the PR: https://github.com/ecosia/go-prometheus-workshop/pull/4/files#diff-04c6e90faac2675aa89e2176d2eec7d8R45

Reply:
oops sorry